packages/eslint-plugin/src/configs/react.js

```diff
@@ -220,6 +220,7 @@ const config = {
     'react/sort-comp': 'off',
     'react/state-in-constructor': 'off',
     'react/static-property-placement': 'off',
+    'react/require-default-props': 'off',
```

Comment:
prior to  moving airbnb rules to exact/single version policy default-props was no properly triggered. with this PR it started to work and failed basically all around the monorepo - thus turning it off as our component composition model is more advance than this rule capabilities